### PR TITLE
Update typo and remove a script don't need

### PR DESCRIPTION
**Pull request by kuanfandevops**

### CODE REVIEW

File: openshift/templates/maintenance-page/README.md [hunk 1]
```diff
@@ -1,169 +1,45 @@
...
 * mem-tfrs-prod
 * mem-tfrs-dev
 * mem-tfrs-test
 
-For the sake of simplicity all examples will use mem-tfrs-test and be run on OS X.
-
-1. ##### Enable/Disable by Script
-
-    Maintenance mode on.
+1. ##### Build Miantanence Page Image
```
Comment (amichard) [inline openshift/templates/maintenance-page/README.md]: Seems like a typo for Maintenance

File: openshift/templates/maintenance-page/README.md [hunk 2]
```diff
@@ -1,169 +1,45 @@
...
 
     ```
-    ./maintenance.sh mem-tfrs-test on
+    oc project mem-tfrs-tools
+    oc process -f ./maintenance-page.bc.json | oc create -f -
     ```
 
-    Maintenance mode off.
+2. ##### Deploy Miantanence Page Image
```
Comment (amichard) [inline openshift/templates/maintenance-page/README.md]: Same here.

File: openshift/templates/maintenance-page/README.md (hunk 1 above)
Comment (kuanfandevops) [reply openshift/templates/maintenance-page/README.md]: corrected

File: openshift/templates/maintenance-page/README.md (hunk 2 above)
Comment (kuanfandevops) [reply openshift/templates/maintenance-page/README.md]: corrected

**Pull request merged**